docs/source/tutorials/object_detection.ipynb

```diff
@@ -119,10 +119,21 @@
    "outputs": [],
    "source": [
     "import pickle\n",
-    "\n",
-    "from cleanlab.object_detection.rank import get_label_quality_scores, issues_from_scores\n",
+    "import numpy as np\n",
+    "from cleanlab.internal.object_detection_utils import calculate_bounding_box_areas\n",
```

Comment:
move this import to lower in the Exploratory data analysis section. Normally we don't like to do this, but in this case, we really shouldn't be encouraging users to important these methods (eg. users who only want to use ObjectLab shouldn't ever have to import this).